tests/integration_tests/Test/PM/Integration/CustomPhase.cls

```diff
@@ -0,0 +1,17 @@
+Class Test.PM.Integration.CustomPhase Extends Test.PM.Integration.Base
+{
+
+Parameter TargetModuleName As STRING = "custom-phase-without-lifecycle";
+
+Method TestCustomPhaseWithoutLifecycle()
+{
+    Set tModuleDir = ..GetModuleDir(..#TargetModuleName)
+    Set tSC = ##class(%IPM.Main).Shell("load -verbose " _ tModuleDir)
+    Do $$$AssertStatusOK(tSC,"Loaded module successfully")
+    Set tSC = ##class(%IPM.Main).Shell(..#TargetModuleName _ " greeting -only")
```

Comment:
Nit: Would be good to capture the output of the call to confirm it wrote the correct message i.e. the phase actually ran.
Also, can add tests for before, after and during for the custom phase